README.md

```diff
@@ -87,18 +64,18 @@ risks introduced are outweighed by the functionalities they support.
 
 ![image](/images/install05.png)
 
-### Authentication with PAT (optional)
-Create a Personal Access Token (PAT) for authentication and save the token value as a repository secret. Review [Personal Access Token (PAT) Requirements and Risks](#personal-access-token-pat-requirements-and-risks) before using a PAT.
-
-1. [Create a Personal Access Token](https://github.com/settings/tokens/new?scopes=public_repo,read:org,read:repo_hook,read:discussion) with the following read permissions:
-    - Note: `Token for OSSF Scorecard Action - myorg/myrepo` (Note: replace `myorg/myrepo` with the names of your organization and repository so you can keep track of your tokens.)
-    - Expiration: `No expiration`
-    - Scopes:
-        * `repo > public_repo`                  Required to read [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) settings. **Note**: for private repositories, you need scope `repo`.
-        * `admin:org > read:org`                Optional: not used in current implementation.
-        * `admin:repo_hook > read:repo_hook`    Optional: needed for the experimental [Webhook](https://github.com/ossf/scorecard/blob/main/docs/checks.md#webhooks) check.
-        * `write:discussion > read:discussion`  Optional: not used in current implementation.
-
+### Authentication with Fine-grained Personal Access Token (optional)
+Create a fine-grained Personal Access Token (PAT) for authentication and save the token value as a repository secret.
+
+1. [Create a fine-grained Personal Access Token](https://github.com/settings/personal-access-tokens/new) with the following settings:
+    - Token name: `OpenSSF Scorecard Action - $USER_NAME/$REPO_NAME>` (Note: replace `$USER_NAME/$REPO_NAME` with the names of your organization and repository so you can keep track of your tokens.)
+    - Expiration: Set `Custom` and then set the date to exactly a year in the future (the maximum allowed)
```

Comment:
I actually ran this test this morning. When I ran the CLI with an expired token I got an immediate error:

```
Error: GetClients: getting OSS-Fuzz repo client: error during InitRepo: repo unreachable: GET https://api.github.com/repos/google/oss-fuzz: 401 Bad credentials []
2023/06/14 21:25:20 error during command execution: GetClients: getting OSS-Fuzz repo client: error during InitRepo: repo unreachable: GET https://api.github.com/repos/google/oss-fuzz: 401 Bad credentials []
```

Unfortunately the API response doesn't mention token expiry, it's a simple:

```json
{
  "message": "Bad credentials",
  "documentation_url": "https://docs.github.com/rest"
}
```

Reply:
Should probably be opened as an issue on the scorecard repo if it hasn't already. I know it's come up in the past that it can be confusing that the "bad credentials" bit is hidden in a mess of error strings.